Modules/Loadable/Transforms/SubjectHierarchyPlugins/qSlicerSubjectHierarchyTransformsPlugin.cxx

```diff
@@ -919,12 +919,25 @@ void qSlicerSubjectHierarchyTransformsPlugin::showViewContextMenuActionsForItem(
   d->showResetCenterOfTransformationForTransformedNodesActions(transformNode);
 
   vtkMRMLTransformDisplayNode* displayNode = transformNode ? vtkMRMLTransformDisplayNode::SafeDownCast(transformNode->GetDisplayNode()) : nullptr;
+
+  bool wasBlocked = d->ToggleInteractionAction->blockSignals(true);
```

Comment:
Is there a specific reason for not using QSignalBlocker?

Reply:
No specific reason, just force of habit. I've updated it to use QSignalBlocker.